qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/ls/template/TemplateFileTextDocumentService.java

```diff
@@ -100,6 +104,10 @@ public TemplateFileTextDocumentService(QuteLanguageServer quteLanguageServer, Sh
 	@Override
 	public void didOpen(DidOpenTextDocumentParams params) {
 		QuteOpenedTextDocument document = (QuteOpenedTextDocument) documents.onDidOpenTextDocument(params);
+		if (!hasOpenedAQuteDocument) {
+			hasOpenedAQuteDocument = true;
+			getLanguageClient().telemetryEvent(new TelemetryEvent(QuteTelemetryConstants.FILE_OPENED, new HashMap<>()));

```

Comment:
`QuteTelemetryConstants` seems to be missing.

Reply:
Fixed.